### PR TITLE
Fix message for kubeconfig, only log for runc when necessary

### PR DESCRIPTION
**Pull request by dongsupark**
Print args only when `logPath` (`$KUBE_SPAWN_RUNC_LOG_PATH`) is explicitly set. Otherwise the logs will appear at the top of every pod's log.

At the end of init command, print out message for setting `$KUBECONFIG` based on real lookups: first try `$PWD/.kube-spawn/default/kubeconfig`, then `$GOPATH/src/github.com/kinvolk/kube-spawn/.kube-spawn/default/kubeconfig`, then `$HOME/go/src/github.com/kinvolk/kube-spawn/.kube-spawn/default/kubeconfig`.

Fixes https://github.com/kinvolk/kube-spawn/issues/104 https://github.com/kinvolk/kube-spawn/issues/111

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -70,6 +86,26 @@ func GetValidCniPath(inGoPath string) (string, error) {
 	return cniPath, nil
 }
 
+func GetValidKubeConfig() string {
+	kcPath := filepath.Join(pwd, kcRelPath)
```
Comment (iaguis) [inline pkg/utils/utils.go]: For some reason this doesn't work for me. It prints:

```
2017/08/08 11:54:37 Note: For kubectl to work, please set $KUBECONFIG to .kube-spawn/default/kubeconfig
```

which is not good because setting `$KUBECONFIG` to that will only make kubectl work if I'm in the kube-spawn directory.

Can you use `os.Getwd()` instead?

File: cmd/kube-spawn/init.go [hunk 2]
```diff
@@ -94,5 +95,5 @@ func doInit() {
 		}
 	}
 
-	log.Println("Note: For kubectl to work, please set $KUBECONFIG to $HOME/go/src/github.com/kinvolk/kube-spawn/.kube-spawn/default/kubeconfig.")
+	log.Printf("Note: For kubectl to work, please set $KUBECONFIG to %s\n", utils.GetValidKubeConfig())
```
Comment (iaguis) [inline cmd/kube-spawn/init.go]: Why the change from Printf to Println?

Comment (dongsupark) [inline cmd/kube-spawn/init.go]: Just a matter of taste, I just think that `Printf("This is %s\n", str)` look better than `Println("This is " + str)`. Doesn't matter actually. Will change it to Println.

Comment (iaguis) [inline cmd/kube-spawn/init.go]: Oh I see, didn't realize there was an argument there. You can keep the `Printf`

File: pkg/utils/utils.go (hunk 1 above)
Comment (dongsupark) [inline pkg/utils/utils.go]: @iaguis Updated. Now it uses `os.Getwd()` to get the current working directory, and prints out the whole abspath.
